server/routes/detail.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const router = express.Router();
+const Detail = require('../../db/Detail.js');
```

Comment:
This is kind of nitpicky, but all other files are not capitalized, but this one is. Chose one or the other.

Reply:
Fixed!